Resources/Prototypes/Recipes/Lathes/tools.yml

```diff
@@ -141,6 +141,24 @@
     Steel: 800
     Glass: 300
 
+- type: latheRecipe
+  id: trayScanner
```

Comment:
```suggestion
  id: TrayScanner
```

Reply:
entity prototype ID isnt capitalized (though it should be) but might as well not make it worse

Reply:
Capitalization adjusted, thank you.